rhizome/host/lib/vm_setup.rb

```diff
@@ -470,43 +470,43 @@ def download_boot_image(boot_image, custom_url: nil, ca_path: nil)
 
     download = urls.fetch(boot_image) || custom_url
     image_path = vp.image_path(boot_image)
```

Comment:
Nice, thanks for the commit message and breaking this out; reviewing it with ?w=1 in github was great.